firebase-vscode/src/workflow.ts

```diff
@@ -145,26 +145,32 @@ export function setupWorkflow(
 ) {
   extensionContext = context;
 
-  // Get user-defined VSCode settings.
-  const workspaceConfig = workspace.getConfiguration(
-    'firebase',
-    vscode.workspace.workspaceFolders[0].uri
-  );
-  const shouldDebug: boolean = workspaceConfig.get('debug');
+  var shouldDebug: boolean;
+  if (vscode.workspace.workspaceFolders) {
+    // Get user-defined VSCode settings.
+    const workspaceConfig = workspace.getConfiguration(
+      'firebase',
+      vscode.workspace.workspaceFolders[0].uri
```

Comment:
nit: shall we display a warning if multiple folders are opened? For example, "the Firebase Extension will use config from the first folder" or something